client/app/pages/queries-list/index.js

```diff
@@ -43,16 +75,49 @@ class QueriesListCtrl {
       });
     }
 
-    this.paginator = new LivePaginator(queriesFetcher, { page });
+    this.paginator = new LivePaginator(
+      queriesFetcher,
+      {
+        page,
+        itemsPerPage: this.pageSize,
+        orderByField: pageOrder,
+        orderByReverse: pageOrderReverse,
+        params: this.parameters,
```

Comment:
does `this.parameters` have a value at this point? (I see where `update()` assigns to it later.)

Reply:
It's a bit of a kerfuffle, I'll clean this up while porting this to https://github.com/getredash/redash/pull/2573